lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -46,6 +46,8 @@ var edx = edx || {};
         _getPaymentButtonText: function(processorName) {
             if (processorName.toLowerCase().substr(0, 11)=='cybersource') {
                 return gettext('Pay with Credit Card');
+            } else if (processorName.toLowerCase()=='paypal') {
```

Comment:
Minor: Set `processorName = processorName.toLowerCase();` at the beginning of the function.